src/sexp.ml

```diff
@@ -58,6 +60,8 @@ let rec block = function
       List [Atom "html"; Atom s]
   | Link_def {label; destination; _} ->
       List [Atom "link-def"; Atom label; Atom destination]
+  | Tag_block {tag; content; _} ->
+      List [Atom "tag"; Atom tag; List (Atom "list-item" :: List.map block content)]
```

Comment:
Why is there a `list-item` here ? Shouldn't this be
```
List [Atom "tag"; Atom tag; List (List.map block content)]
```

Reply:
Okay my bad, I never use sexp before and simply refered to the use above, I'll take that into note